tools/build_executable/LinuxEngine.ini

```diff
@@ -0,0 +1,23 @@
+;
+; Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+;
+
+; NOTE: This custom configuration file is inserted into public release builds by
+; spear/tools/build_executable/build_executable.py. This file's original location in
+; the spear codebase is spear/tools/build_executable. We insert this file into public
+; release builds in order to modify the default map. Doing so enables a user to
+; double-click on a SpearSim executable and navigate around our Kujiale test scene
```

Comment:
Should we say 'Kujiale *test* scene'?

Reply:
I don't mind the current wording of this. We could also say "our kujiale_0000 scene", but I think the current wording is a slightly higher-level and more accessible.